File: dale-joshua/maximum-subarray-sum.js
var maxSequence = function(arr){
  let maxSum = 0
    let currentSum = 0
    for(let i = 0; i < arr.length; i++){ 
        currentSum = Math.max(arr[i], currentSum + arr[i])
        maxSum = Math.max(currentSum, maxSum)
      console.log(currentSum, maxSum)
        
    }
  if(maxSum < 0) {
    return 0
  }
  else {
    return maxSum
    }

File: mekonnen-naod/Maximum-subarray-sum.py
var maxSequence = function(arr){

let maxSum = 0;
let subSum = 0

for(let i=0; i<arr.length; i++){
  
  subSum = Math.max(arr[i], subSum + arr[i])
  maxSum = Math.max(subSum, maxSum)
}


return maxSum

}
